### ISSUE TITLE
[OneNote] Remember me option

### ISSUE THREAD

**Issue by p3rid0t** [obsidianmd/obsidian-importer#117]
While working on the OneNote Importer, I found it pretty annoying to constantly sign in. That made me think of adding an option which allows users to remember their account details.

## Implementation Details
Adding the option should be relatively straightforward. However, there is a pretty big concern regarding the storage of access/refresh tokens for user accounts. These tokens are sensitive pieces of information, and it's important to ensure they are stored securely to prevent other (possibly malicious) plugins from accessing them. 

It's why I suggest implementing this option as an opt-in feature rather than making it the default behavior. This way, users who frequently perform imports can choose to enable it for their convenience, while most who prefer more security are not affected.

This could possibly be added as a core feature of the Importer, rather than just adding it to the OneNote Importer, so that other devs who may want to use OAuth as well can easily & safely implement it.

## TL;DR
Some users may need to frequently import notes. To make it easier, we could add an ***opt-in*** toggle for remembering their account details.

**Comment by lishid** [comment]
That would be pretty nice to have - I think the OAuth token still has an expiry time so we should probably test if it still works when the user is using a remembered token.
